onnx_caffe2/backend.py

```diff
@@ -461,6 +461,19 @@ def _create_reshape(cls, n):
         return c2_op
 
     @classmethod
+    def _direct_initialize_parameters(cls, initializer, ws):
+        for tp in initializer:
+            ws.FeedBlob(tp.name, onnx.numpy_helper.to_array(tp))
+
+    @classmethod
+    def _direct_initialize_inputs(cls, inputs, initialized, ws):
+        for value_info in inputs:
+            if value_info.name in initialized:
+                continue
+            shape = list(d.dim_value for d in value_info.type.tensor_type.shape.dim)
```

Comment:
inputs are probably small. But doing `ws.RunOperatorOnce('GivenTensorFill', ...))` might be even faster